index.js

```diff
@@ -46,11 +46,24 @@ class Plugin extends MiniAccountsPlugin {
     this._claimInterval = opts.claimInterval || util.DEFAULT_CLAIM_INTERVAL
     this._store = new StoreWrapper(opts._store)
     this._txSubmitter = createSubmitter(this._api, this._address, this._secret)
+    this._currencyScale = opts.currencyScale || 6
 
     this._channelToAccount = new Map()
     this._accounts = new Map()
   }
 
+  xrpToBase (amount) {
+    return new BigNumber(amount)
+      .mul(Math.pow(10, this._currencyScale))
+      .toString()
+  }
+
+  baseToXrp (amount) {
+    return new BigNumber(amount)
+      .div(Math.pow(10, this._currencyScale))
+      .toFixed(6, BigNumber.ROUND_UP)
```

Comment:
Wouldn't it be better to round down and just the extra as unsettled balance?

Reply:
The thing is, these plugins are emitting the amount of money they got in the base scale. If you round down it means the other side needs to round the amount they received down and then emit it. For some amounts this means a sendMoney call will send 0. Technically the error would still be bounded at one drop, but I think it's more confusing to work with if the sendMoney amount doesn't match the amount emitted by the other side